### ISSUE TITLE
`yarn add` installs a empty directory in `node_modules`

### ISSUE THREAD

**Issue by cbarraford** [TrustWallet/web-core#3]
when i did `yarn add @trustwallet/walletconnect` all that it installed was an empty directory into `node_modules`, except a `package.json` file. 

```
{
  "name": "@trustwallet/walletconnect",
  "version": "0.0.11",
  "description": "Trust WalletConnect extensions",
  "main": "lib/index.js",
  "directories": {
    "lib": "lib",
    "test": "test"
  },
  "files": [
    "lib",
    "src"
  ],
  "scripts": {
    "clean": "rm -rf lib",
    "build": "webpack",
    "test": "mocha --require ./babel-polyfill.js test/**/*.spec.ts"
  },
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/trustwallet/web-core.git"
  },
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/trustwallet/web-core/issues"
  },
  "homepage": "https://github.com/trustwallet/web-core#readme",
  "keywords": [],
  "publishConfig": {
    "access": "public"
  },
  "devDependencies": {
    "@babel/cli": "^7.1.2",
    "@babel/core": "^7.1.2",
    "@babel/node": "^7.0.0",
    "@babel/polyfill": "^7.0.0",
    "@babel/preset-env": "^7.1.0",
    "@babel/preset-typescript": "^7.1.0",
    "@babel/register": "^7.0.0",
    "@types/chai": "^4.1.6",
    "@types/mocha": "^5.2.6",
    "chai": "^4.1.2",
    "isomorphic-fetch": "^2.2.1",
    "mocha": "5.2.0",
    "ts-loader": "^5.3.3",
    "typescript": "^3.1.3",
    "webpack": "^4.28.3",
    "webpack-cli": "^3.2.1"
  },
  "dependencies": {
    "@walletconnect/browser": "^1.0.0-beta.31",
    "@walletconnect/core": "^1.0.0-beta.31",
    "@walletconnect/types": "^1.0.0-beta.31"
  },
  "gitHead": "9ce87562531165a939631ecc0db162984392def3"
}
```

**Comment by leoneparise** [comment]
@cbarraford the issue is now fixed in version `0.0.12`.

**Comment by cbarraford** [comment]
thanks @leoneparise. i'll verify it on monday.
